src/lib/Metadata/Finalizers/BaseMembersFinalizer.cpp

```diff
@@ -30,53 +30,57 @@ inheritBaseMembers(
     RecordInterface const& base,
     AccessKind const A)
 {
+    auto&& members = allMembers(derived);
```

Comment:
Isn't `members` a value type with const-reference semantics? Is this a reference to a temporary we're relying on the compiler to keep alive?

If so, wouldn't 

```
auto members = allMembers(derived);
```

mean the same thing here since this is just a view?